app/serializers/admin_detailed_user_serializer.rb

```diff
@@ -42,6 +42,7 @@ class AdminDetailedUserSerializer < AdminUserSerializer
   has_one :silenced_by, serializer: BasicUserSerializer, embed: :objects
   has_one :tl3_requirements, serializer: TrustLevel3RequirementsSerializer, embed: :objects
   has_many :groups, embed: :object, serializer: BasicGroupSerializer
+  has_many :user_associated_accounts, serializer: UserAssociatedAccountSerializer, embed: :objects
```

Comment:
I think it would be nice to have symmetry between the create/update API, and this 'show' API. So rather than a `user_associated_accounts` key, I wonder if we should have something like this:

```
external_ids: {
  "facebook": "blah",
  "oidc": "externalid2",
}
```
What do you think? 🤔